From 8ab2de318d55c9d0987ffae7d9b94b365af732c1 Mon Sep 17 00:00:00 2001
From: Alexandre Janniaux <ajanni@videolabs.io>
Date: Wed, 14 Sep 2022 16:29:22 +0200
Subject: [PATCH 1/2] icy: include stdlib.h and stdio.h

malloc, free, and snprintf had no valid previous declaration, making the
build fail with clang.
---
 src/icy.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/icy.c b/src/icy.c
index 8e83552..4de3c14 100644
--- a/src/icy.c
+++ b/src/icy.c
@@ -24,6 +24,8 @@
 #endif
 
 #include <stdbool.h>
+#include <stdlib.h> /* for free() and malloc() */
+#include <stdio.h> /* for snprintf() */
 #include <string.h>
 
 #include <shout/shout.h>
-- 
GitLab
